ledger/participant-integration-api/src/main/scala/platform/store/dao/ParametersTable.scala

```diff
@@ -56,9 +56,11 @@ private[store] object ParametersTable {
     )
 
   def getParticipantId(connection: Connection): Option[ParticipantId] =
-    SQL"select #$ParticipantIdColumnName from #$TableName".as(ParticipantIdParser.single)(
-      connection
-    )
+    SQL"select #$ParticipantIdColumnName from #$TableName"
```

Comment:
this changes the behavior. was it delibare? why is this needed?

Reply:
Before: crash if parameter table is empty
After: return `None` if parameter table is empty

This makes `getParticipantId` behave the same as `getLedgerId`. This was changed for sandbox initialization - SqlLedger reads the ledger and participant id before attempting to initializing the database.

Reply:
makes sense